assets/src/dashboard/components/dropdown/stories/index.js

```diff
@@ -93,24 +93,47 @@ export const _transparent = () => {
   );
 };
 
+const categoryDemoData = [
+  {
+    label: <span>{'All Categories'}</span>,
+    value: 'all',
+    selected: true,
+  },
+  { label: 'Arts and Crafts', value: 'arts_crafts' },
+  { label: 'Beauty', value: 'beauty' },
+  { label: 'Cooking', value: 'cooking' },
+  { label: 'News', value: 'news' },
+  { label: 'Sports', value: 'sports' },
+  { label: 'News', value: 'news' },
+  { label: 'UNCLICKABLE', value: false },
```

Comment:
For these we can still use the `__()` Wordpress i18n helper

Reply:
even though it's just dummy content in storybook?

Reply:
(updated)

Reply:
Yeah, I'm not sure we need to translate storybook text... I don't see "UNCLICKABLE" being translated any time soon :D